src/main/resources/crafter/studio/studio-config.yaml

```diff
@@ -191,6 +191,24 @@ studio.security.ldap.userAttribute.siteId: crafterSite
 studio.security.ldap.userAttribute.groupName: crafterGroup
 # Global admin group
 studio.security.global.adminGroup: crafter-admin
+# Authentication via headers enabled
+studio.authentication.headers.enabled: false
+# Authentication header for secure key
+studio.authentication.headers.secureKeyHeader: saml_secure_key
+# Authentication headers secure key
```

Comment:
Update this comment to indicate that this should be a secret key that you expect to match with the headers and typically this placed in the header by the authentication agent, e.g. Apache mod_mellon